lib/providers/httpTransaction/index.js

```
@@ -216,28 +216,27 @@ HttpTransaction.prototype.onServerReceive = function (data) {
   })
 }
 
-HttpTransaction.prototype.report = function (data) {
+HttpTransaction.prototype.report = function (name, userData) {
```

If the first param is not a string it should throw an error